model/patch_lifecycle.go

```diff
@@ -575,6 +576,10 @@ func FinalizePatch(ctx context.Context, p *patch.Patch, requester string, github
 		if err != nil {
 			return nil, errors.Wrapf(err, "inserting version '%s'", patchVersion.Id)
 		}
+		_, err = CreateManifest(*patchVersion, project, projectRef, settings)
```

Comment:
this is being done within a transaction; maybe we should pull out the "insert" part from the "create" part so that it can be created outside the transaction and inserted using the sessCtx?

Reply:
Sure!